solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -275,6 +276,18 @@ private void dispatch(
       }
     } finally {
       call.destroy();
+      SolrCore core = call.getCore();
+      if (core != null) {
+        int coreRefCount = core.getOpenCount();
+        if (coreRefCount > 1) {
+          if (log.isInfoEnabled())
```

Comment:
that will log this message whenever core is serving more than one request? if so it might be huge?

Reply:
Yes, this could happen but I would assume it's still relative rare that a given core is accessed by multiple requests concurrently?

The log line itself is pretty short, so my hope is that it shouldn't burden the log too much.

Reply:
Also crunched some number from BQ on one hour of log in prod `fullstoryapp.stackdriver_logs.vm_solr_20230910_H18`.

Total log entries: 126,535,028

To roughly estimate the extra entries caused by this, I looked at the number of "Core request" completion message logged by `SolrCore#execute` : 22,419,120. Every incoming core request should log that completion message exactly once.

It is true that our log added here has a bigger scope than "Core request" handled by `SoreCore#execute`, for example our change could be reached for admin request (core status?) that uses a core as well, however, I would consider MOST of the requests on prod comes in the form that triggers `SoreCore#execute` (query etc).

So in the worst case scenario, if all cores have concurrent requests, then it could add around 20% more log lines. However, I see that highly unlikely. I would be surprised if it adds more than 1m+ entry per hour 😓